shared/elements/gaia_grid/js/grid_dragdrop.js

```diff
@@ -627,24 +650,19 @@
       }
 
       this.inEditMode = false;
+      this.cancelLongPressTimeout();
       this.container.classList.remove('edit-mode');
       document.body.classList.remove('edit-mode');
       this.gridView.element.dispatchEvent(new CustomEvent('editmode-end'));
       document.removeEventListener('visibilitychange', this);
-      this.removeDragHandlers();
       this.gridView.render();
     },
 
-    removeDragHandlers: function() {
-      this.container.removeEventListener('touchmove', this);
-      this.container.removeEventListener('touchend', this);
-      window.removeEventListener('touchcancel', this);
-    },
-
-    addDragHandlers: function() {
-      this.container.addEventListener('touchmove', this);
-      this.container.addEventListener('touchend', this);
-      window.addEventListener('touchcancel', this);
+    cancelLongPressTimeout: function() {
+      if (this.longPressTimeout) {
```

Comment:
Never thought about it before - I don't think you can have a timeout that tests true, but I'm not certain - that's definitely safer and I think it reads better too, will do.